docs/source/api_reference/metalearning.rst

```diff
@@ -19,3 +19,16 @@ MAML
 
 .. autoclass:: deepchem.metalearning.MAML
   :members:
+
+MetaLearner
+----------------
+This is the abstract superclass for metalearning algorithms.
+
+.. autoclass:: deepchem.metalearning.torch_maml.MetaLearner
+  :members:
+
+MAML
```

Comment:
Mention this is the torch implementation and the earlier implementation is the TensorFlow implementation

Reply:
Done

Reply:
Let's add the new MAML abstract class to the shared documentation